app/views/super_admin/super_admin.py

```diff
@@ -35,6 +36,7 @@ def index_view():
     rejected_sessions = get_count(DataGetter.get_all_rejected_sessions())
     draft_sessions = get_count(DataGetter.get_all_draft_sessions())
     email_times = DataGetter.get_email_by_times()
+    total_users=registered_users + unverified_users
```

Comment:
Instead of doing the total here, storing it in a variable and then passing to the template and referring it there, it will be easier/simpler to do the total directly in the template